### PR TITLE
CLOUD-476 add dbaas build infrastructure

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: dbaas/jenkins/percona-dbaas-cli-autobuild.groovy [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            }
+        }
+        stage('Build dbaas source rpm') {
+            steps {
+                sh 'sg docker -c "./build/bin/build-dbaas-srpm"'
+                stash includes: 'results/srpm/*.src.*', name: 'rpms'
+                uploadRPM()
+            }
+        }
+        stage('Build adaas rpm') {
```
Comment (delgod) [inline dbaas/jenkins/percona-dbaas-cli-autobuild.groovy]:
```suggestion
        stage('Build dbaas rpm') {
```

File: dbaas/jenkins/percona-dbaas-cli-autobuild.groovy [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            steps {
+                sh 'sg docker -c "./build/bin/build-dbaas-rpm centos:7"'
+                sh 'sg docker -c "./build/bin/build-dbaas-rpm centos:6"'
+                stash includes: 'results/rpm/*.rpm', name: 'rpms'
+                uploadRPM()
+            }
+        }
+        stage('Build adaas source deb') {
```
Comment (delgod) [inline dbaas/jenkins/percona-dbaas-cli-autobuild.groovy]:
```suggestion
        stage('Build dbaas source deb') {
```

File: dbaas/jenkins/percona-dbaas-cli-autobuild.groovy [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        stage('Build dbaas source rpm') {
+            steps {
+                sh 'sg docker -c "./build/bin/build-dbaas-srpm"'
+                stash includes: 'results/srpm/*.src.*', name: 'rpms'
+                uploadRPM()
+            }
+        }
+        stage('Build adaas rpm') {
+            steps {
+                sh 'sg docker -c "./build/bin/build-dbaas-rpm centos:7"'
```
Comment (delgod) [inline dbaas/jenkins/percona-dbaas-cli-autobuild.groovy]: add centos8
```suggestion
                sh 'sg docker -c "./build/bin/build-dbaas-rpm centos:8"'
                sh 'sg docker -c "./build/bin/build-dbaas-rpm centos:7"'
```

File: vars/sync2ProdDBaas.groovy [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                            done
+
+                            for dist in `ls -1 debian`; do
+                                for deb in `find debian/\${dist} -name '*.deb'`; do
+                                    repopush --remove-package --gpg-pass ${SIGN_PASSWORD} --package \${deb} --verbose --component ${DESTINATION} --codename \${dist} --repo-path /srv/repo-copy/tools/apt
+                                done
+
+                                # source deb
+                                #for dsc in `find ../source -name '*.dsc'`; do
+                                #    repopush --remove-package --gpg-pass ${SIGN_PASSWORD} --package \${dsc} --verbose --component ${DESTINATION} --codename \${dist} --repo-path /srv/repo-copy/tools/apt
```
Comment (delgod) [inline vars/sync2ProdDBaas.groovy]: why not?

File: dbaas/jenkins/percona-dbaas-cli-autobuild.groovy (hunk 3 above)
Comment (hors) [reply dbaas/jenkins/percona-dbaas-cli-autobuild.groovy]: done

File: dbaas/jenkins/percona-dbaas-cli-autobuild.groovy (hunk 2 above)
Comment (hors) [reply dbaas/jenkins/percona-dbaas-cli-autobuild.groovy]: done

File: dbaas/jenkins/percona-dbaas-cli-autobuild.groovy (hunk 1 above)
Comment (hors) [reply dbaas/jenkins/percona-dbaas-cli-autobuild.groovy]: done

**Pull request merged**